src/AppBundle/Resources/views/themes/app/pagelayout_variables.html.twig

```diff
@@ -24,6 +24,14 @@
     {% endif %}
 {% endif %}
 
+{# CURRENT LOCATION PATH ID ARRAY #}
+{% set current_location_path = [] %}
+{% if app.request.attributes.get('_route') == 'ez_urlalias' and app.request.attributes.has('view') %}
```

Comment:
This condition is redundant. This template is used only in full views, so no need for route check. The one below is enough.

Reply:
@emodric Are you sure? I have set this condition because we saw a lot of unexpected crashes due to it.
To give some examples:
1. Try removing the condition and then do a content preview in ngadminui, I think that it crashes without this condition. 
2. custom symfony routes where there was a "content" or "location" object available, but it was not an eZ content or eZ location, just an unfortunate name match.
3. Frontend content editing - you get the content object, but in version 1, you do not have a location yet.

 This makes sure we are rendering an eZ full view and serves as a safety measure .

Reply:
As for 1), nope, it does not crash. But even if it did, we'd have to solve it some other way, since I'd expect that JS active state works even when the route is not `ez_urlalias`, e.g. eZ admin preview or Netgen Layouts preview.

2) There are other places where we don't check this. `content` and `location` are always assumed to come from eZ and the whole `pagelayout.html.twig` and `pagelayout_variables.html.twig` are geared towards it.

3) This case will be handled by `{% if location is defined %}` will it not? (`location|default(null) is not empty` check is also redundant, I think, as per my reply on your second point)